### ISSUE TITLE
Firefox issue

### ISSUE THREAD

**Issue by kaarleenzz** [vladmalik/pasteimage#4]
"Doesn't work in FF if the focus is inside another text area (hypothesis: paste event happens before the plugin can refocus on the pastecatcher DIV). Need a way to delay paste event."

Have you managed to find a way to work around this? 

I have multiple places on my page where I can paste the image assign the source to different input depending on which one is selected (focused).

**Comment by vladmalik** [comment]
No, it wasn't an issue for my purposes. I doubt if the paste event can be intercepted. I know people use flash plugins to control clipboard behavior. Maybe look into that. I don't know. Other than that, I'd maybe create a separate paste window where paste can happen. So you click to trigger paste window, so when you paste there is just one place where the image can go. Let me know if you solve this and feel free to contribute code.
